dj_cqrs/admin.py

```diff
@@ -0,0 +1,44 @@
+class CQRSAdminMasterSyncMixin:
+    """
+    Mixin that includes, on AdminModel, a custom action that allows syncronization of master's
+    model items from Django Admin page.
+    """
+
+    def get_actions(self, request):
+        """
+        Overrided method from AdminModel class, to include in actions list the sync method.
+        """
+        if self.actions is not None:
+            self.actions = self.actions + ['sync_items']
+        return super().get_actions(request)
+
+    def _cqrs_sync_queryset(self, queryset):
+        """
+        This function can be used to make any adjustment on the queryset before sending the sync
+        sign.
+        :param queryset: Original queryset
+        :type queryset: Queryset
+        :return: Updated queryset
+        :rtype: Queryset
+        """
+        return queryset
+
+    def sync_items(self, request, queryset):
+        """
+        This method syncronized the selected items from the admin page. It's register as a
```

Comment:
This method synchronizes selected items from the Admin Page. It is registered as a custom action in Django Admin